scripts/tests/sanitycheck/test_testinstance.py

```diff
@@ -43,14 +43,13 @@ def test_check_build_or_run(class_testsuite, monkeypatch, all_testcases_dict, pl
     testcase.slow = slow
 
     testinstance = TestInstance(testcase, platform, class_testsuite.outdir)
-    testinstance.check_build_or_run(build_only, slow, device_testing, fixture)
+    run = testinstance.check_build_or_run(build_only, slow, device_testing, fixture)
     b, r = expected
-    assert testinstance.build_only == b
```

Comment:
Since `b` is no longer used in this test maybe it will be cleaner to also delete it from `TESTDATA_1` and just use  `r = expected`?